api/actions/serializers.py

```diff
@@ -131,6 +137,9 @@ def create(self, validated_data):
         comment = validated_data.pop('comment', '')
         permissions = validated_data.pop('permissions', '')
         visible = validated_data.pop('visible', '')
+
+        if isinstance(target, Registration):
+            target = target.draft_registration.last()
```

Comment:
Didn't we stop using the draft registration as a target, why is this still here?

Reply:
Cause I'm a big dummy.